packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.widgetkey = '';
+  }
+
+  initiateZenDesk() {
+    // Hide custom button
+    this.style.opacity = 0;
+
+    // prevent further if third party script already loaded
+    if (this.abortIfScriptAlreadyLoaded()) return;
```

Comment:
This will never return early because the return value of this method is always `undefined`.